<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timeline</class>
 <widget class="QWidget" name="Timeline">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#frameGlobal {
	background-color: 	transparent;
	border: 					0px solid black;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameGlobal">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,100">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <item>
         <widget class="QPushButton" name="ffButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="icon">
           <iconset resource="../icons/Rekall.qrc">
            <normaloff>:/icons/res_icon_ff.png</normaloff>:/icons/res_icon_ff.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="icon">
           <iconset resource="../icons/Rekall.qrc">
            <normaloff>:/icons/res_icon_play.png</normaloff>
            <normalon>:/icons/res_icon_pause.png</normalon>:/icons/res_icon_play.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="hZoomLabel">
          <property name="text">
           <string>TIME STRETCH</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="hZoom">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="vZoomLabel">
          <property name="text">
           <string>OVERALL SIZE</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="vZoom">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="writeNote">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Write a note</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="TimelineGL" name="gl" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimelineGL</class>
   <extends>QWidget</extends>
   <header>gui/timelinegl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons/Rekall.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>playButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>Timeline</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ffButton</sender>
   <signal>released()</signal>
   <receiver>Timeline</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>writeNote</sender>
   <signal>released()</signal>
   <receiver>Timeline</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>action()</slot>
 </slots>
</ui>
